mimesis/providers/generic.py

```diff
@@ -38,17 +40,18 @@ def __init__(self, *args, **kwargs):
         self._science = Science
         self._code = Code
         self._transport = Transport
-        self.unit_system = UnitSystem()
```

Comment:
The attributes starting with `_` is attributes which should be initialized lazily because they need locales for correct work. Other providers do not need locale for correct working because they contain international data.

Reply:
`File` provider not lazy loaded, but when initialized it create new `Text` provider which use locales.